frontend/device/kobo/device.lua

```diff
@@ -48,6 +48,7 @@ local Kobo = Generic:new{
     -- HW inversion is generally safe on Kobo, except on a few boards/kernels
     canHWInvert = yes,
     home_dir = "/mnt/onboard",
+    canToggleMassStorage = yes,
```

Comment:
no prerrequisites?

Reply:
Nope, once I bump base, the tool is baked in ;).

Reply:
awesome, now I want a new Kobo ;p